designsystem/src/main/java/com/mobven/designsystem/components/common/BigCard.kt

```diff
@@ -0,0 +1,118 @@
+package com.mobven.designsystem.components.common
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
+import com.mobven.components.R
+import com.mobven.designsystem.theme.mainSecondary
+import com.mobven.designsystem.theme.neutralGrayscale70
+import com.mobven.designsystem.theme.neutralGrayscale80
+import com.mobven.designsystem.theme.promoOrange
+
+@Composable
+fun YummyBigCard() {
+    Card(
+        shape = RoundedCornerShape(8.dp),
+        modifier = Modifier.fillMaxWidth(),
```

Comment:
fillmaxwidth'in composable'ın kendisinde olması kullanım alanını daraltabilir. Modifier YummyBigCard içerisinden parametre olarak alınabilir